src/app/media-viewer-wrapper/toolbar-toggles/toolbar-toggles.component.html

```diff
@@ -27,7 +27,7 @@
         </div>
         <div class="govuk-grid-column-full">
           <label for="toggleAnnotations" class="switch govuk-!-margin-right-2">
-            <input type="checkbox" id="toggleAnnotations" [checked]="toggleAnnotations.getValue()" (change)="toggleAnnotations.next($event.target.checked)">
+            <input type="checkbox" id="toggleAnnotations" [checked]="toggleAnnotations.getValue()" (change)="toggleAnnotations.next($event.target.checked)" aria-label="annotate">
```

Comment:
is there still a need for the label?

Reply:
yeah, I don't think the labels are needed
also, I don't think we need to change the wrapper. it's just for demo purposes, so we don't really care if it's accessible or not

Reply:
yeah this has all been raised by @jai-santhosh so speak to him. Also the labels were there prior and when removing them it causes issues with the buttons.